### PR TITLE
feat(language): add v2 service

### PR DESCRIPTION
**Pull request by devbww**
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/12376)

### CODE REVIEW

File: google/cloud/language/CMakeLists.txt [hunk 1]
```diff
@@ -112,7 +112,8 @@ target_include_directories(
 target_compile_options(google_cloud_cpp_language_mocks
                        INTERFACE ${GOOGLE_CLOUD_CPP_EXCEPTIONS_FLAG})
 
-if (BUILD_TESTING AND GOOGLE_CLOUD_CPP_ENABLE_CXX_EXCEPTIONS)
+include(CTest)
+if (BUILD_TESTING)
```
Comment (coryan) [inline google/cloud/language/CMakeLists.txt:116]: The quickstart has a `catch(Status const&)`.  Now I wonder if the `noex` build is compiling the quickstarts.

Comment (coryan) [reply google/cloud/language/CMakeLists.txt:116]: FWIW, I think we should leave the `AND GOOGLE_CLOUD_CPP_ENABLE_CXX_EXCEPTIONS` and fix the build or all the CMake files in a separate PR.

**Review by coryan — APPROVED** [review]
A small nit in the CMake file, otherwise LGTM.

File: google/cloud/language/CMakeLists.txt (hunk 1 above)
Comment (devbww) [reply google/cloud/language/CMakeLists.txt:116]: I messed up the re-basing, so I'm going to merge this and address your comment in a follow-up PR.

**Pull request merged**